pipelines/concourse.yml

```diff
@@ -128,6 +128,30 @@ jobs:
         mode: normal
         alert_type: broke
 
+- name: build-integration-image
```

Comment:
could we call this `build-containerd-integration-image` or something similar. Someone who rolls onto the team might find it extremely confusing that this isn't the image used for our main integration tests

Reply:
if I understood correctly, it's not *only* for testing `containerd` - it'd be used for the whole set of testflight tests, *as well as* `containerd`'s integration ones.

is that right, @kcmannem?

thx!

Reply:
@cirocosta you're right! It is the easiest way we could run both existed test flights and `containerd` together maybe. Any concern on this?

Reply:
@deniseyu good point! But I am worrying whether it is going to confuse people after renaming it to that name, while it is used for both for the whole test flights.
What about `build-containerd-enabled-integration-image`?

Reply:
> @cirocosta [...]  Any concern on this?

ooh, no concerns! I was just raising the "why" I think this was not named after `*-containerd-*` 😁